### PR TITLE
Add LTI component as the Activity to the Bridge's Collection

### PR DESCRIPTION
**Pull request by idegtiarov**
Remove filters from the types of received blocks.
Small style fixes in the module.js file.

### CODE REVIEW

File: bridge_adaptivity/module/static/module/js/module.js [hunk 1]
```diff
@@ -28,12 +28,18 @@
...
             }
         });
-        var typeToIcon = {
-            html: "glyphicon-list-alt",
-            problem: "glyphicon-question-sign",
-            video: "glyphicon-facetime-video",
-            other: "glyphicon-option-horizontal"
-        };
+
+        function typeToIcon(itemType) {
```
Comment (flying-pi) [inline bridge_adaptivity/module/static/module/js/module.js]: please use switch construction

File: bridge_adaptivity/module/static/module/js/module.js [hunk 2]
```diff
@@ -131,7 +137,7 @@
                 $("<span/>")
                     .data("toggle", "tooltip")
                     .attr("title", item.type)
-                    .addClass("badge pull-left glyphicon " + typeToIcon[item.type])
+                    .addClass("badge pull-left glyphicon " + typeToIcon(item.type))
```
Comment (flying-pi) [inline bridge_adaptivity/module/static/module/js/module.js]: Maybe template strings more suitable?

File: bridge_adaptivity/module/static/module/js/module.js (hunk 1 above)
Comment (idegtiarov) [reply bridge_adaptivity/module/static/module/js/module.js]: Nice catch, done.

File: bridge_adaptivity/module/static/module/js/module.js (hunk 2 above)
Comment (idegtiarov) [reply bridge_adaptivity/module/static/module/js/module.js]: Yep, template string would be better to use.

**Pull request merged**